redbot/cogs/alias/alias.py

```diff
@@ -326,6 +326,11 @@ async def _list_global_alias(self, ctx: commands.Context):
 
     @commands.Cog.listener()
     async def on_message_without_command(self, message: discord.Message):
+
+        if message.guild is not None:
+            if await self.bot.cog_disabled_in_guild(self, message.guild):
+                return
```

Comment:
You can ignore this review comment, but `cog_disabled_in_guild()` now accepts `None` for guild
```suggestion
        if await self.bot.cog_disabled_in_guild(self, message.guild):
            return
```